VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/wr-participant-list-shared.component.ts

```diff
@@ -204,9 +204,9 @@ export abstract class WRParticipantStatusListDirective {
         const loggedParticipant = this.conference.participants.find(x => x.id === this.loggedInUser.participant_id);
 
         if (this.loggedInUser.role !== Role.Judge && this.loggedInUser.role !== Role.JudicialOfficeHolder) {
-            return !loggedParticipant.current_room?.label.startsWith('JudgeJOH');
+            return !loggedParticipant.current_room?.label.startsWith('JudgeJOH') && !loggedParticipant.linked_participants.length;
```

Comment:
Personally think this method needs more clarity
// if a user is judicial or a judge, check if the room is a judicial consultation room
// else (the user is not judicial)
// check the room isn't a judicial consultation room